button.js

```diff
@@ -0,0 +1,20 @@
+// document.getElementsByTagName('head')[0].getElementById('stylesheet').href='./website.css';
+
+document.addEventListener("DOCContentLoaded", (e)=>{
+    e.disableDefault();
```

Comment:
I think you meant to write `e.preventDefault()`
But you usually use that alongside a submit event, because submit refreshes the page